inventory.yml.sample

```diff
@@ -176,7 +177,14 @@ all:
 
         http_store:
           ansible_host: 10.40.0.100
+        
+        tftp_host
+          ansible_host: 10.40.0.100
+          tftp_directory: /var/lib/tftpboot/ 
```

Comment:
tftp_host shouldn't be here as well. At least not for the default sample inventory.

Reply:
I think we can keep it here for now, since there are some things besides this that we should move from the sample inventory.
This PR is big enough and this can be done as part of the inventory cleanup.

Reply:
> I think we can keep it here for now, since there are some things besides this that we should move from the sample inventory. This PR is big enough and this can be done as part of the inventory cleanup.

Agree

Reply:
Initially, the sample inventory file was a minimal working example -- something you can pick up and extend if needed, using the inventory docs. It doesn't mean that we can't change the purpose of the file, of course. 

> This PR is big enough and this can be done as part of the inventory cleanup.

I'd argue that it's better to add the required changes directly in this PR, we never know if priorities will remain the same, and so the cleanup could get delayed. Also, removing the host would actually make the PR smaller!

Reply:
@mkochanowski Where would you put the documentation about this feature? it still needs to be documented somewhere (with an example)

Reply:
@natifridman I think the Inventory Docs would be a good place (https://github.com/redhat-partner-solutions/crucible/blob/main/docs/inventory.md). We could add a section about configuring alternative boot options (PXE), and maybe link to it from the main README. What do you think, @nocturnalastro?

Reply:
So I did come check to see if `tftp_host` is required. Turns out it not because ansible will skip playbooks if there is no host found for them. 

So we could remove from the base inventory. 

It defiantly should be documented in `inventory.md`

We are missing some inventory validations for checks if `pxe` is a vendor we should check that `tftp_host` is defined and the required values are in the `dns_host`

Reply:
If we wish to have sample inventory with `tftp_host` we could always have a inventory sample for pxe specificity. Perhaps its time to make a samples dir.

Reply:
Add validation  in the inventory validation playbook that if `tftp_host` is defined in the inventory. That it should be equal to the `dns_host`. This should also be documented

Reply:
Done